lib/decidim/reporting_proposals/component.rb

```diff
@@ -59,6 +59,7 @@
     settings.attribute :proposal_wizard_step_4_help_text, type: :text, translated: true, editor: true
     settings.attribute :unanswered_proposals_overdue, type: :integer, default: Decidim::ReportingProposals.unanswered_proposals_overdue
     settings.attribute :evaluating_proposals_overdue, type: :integer, default: Decidim::ReportingProposals.evaluating_proposals_overdue
+    settings.attribute :allow_admins_to_hide_proposals, type: :integer, default: Decidim::ReportingProposals.allow_admins_to_hide_proposals
```

Comment:
```suggestion
```

I don't think it is necessary to have an option in the component to allow admins hide proposals, in the end admins can check this or not, so it is not really necessary.